ibis/expr/operations/relations.py

```diff
@@ -71,6 +71,13 @@ class UnboundTable(PhysicalTable):
     schema = rlz.instance_of(sch.Schema)
     name = rlz.optional(rlz.instance_of(str), default=genname)
 
+    # TODO(kszucs): something like a HasName trait could be useful here
```

Comment:
Isn't `HasName` implemented elsewhere in this PR?

Reply:
Removed the comment. Going to take a look later.